packages/shared/routes/dashboard/Dashboard.svelte

```diff
@@ -359,9 +359,9 @@
         pollLedgerDeviceStatus(false, LEDGER_STATUS_POLL_INTERVAL)
     }
 
-    $: if ($accountsLoaded && $viewableAccounts.length) {
+    $: if ($accountsLoaded) {
         // TODO: persist last selected account
-        setSelectedAccount($viewableAccounts[0]?.id)
+        setSelectedAccount(get(viewableAccounts)?.[0]?.id ?? null)
```

Comment:
This makes the statement only reactive to changes in accountsLoaded, and not in viewableAccounts, right?

Reply:
Yes, making it reactive on `viewableAccounts` means that each time that changes (eg: rename it, receive funds, send funds...) the first account was getting selected, which is not the ideal behavior. With this change its only set once.